### PR TITLE
Migrate UI in a dedicated repository

### PR DESCRIPTION
**Pull request by Mogztter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,31 +6,28 @@
...
+    "build:labs": "antora --fetch --stacktrace netlify-labs-docs.yml",
+    "build:kb": "antora --fetch --stacktrace --generator @neo4j/developer-site-generator netlify-kb.yml",
+    "build:training": "antora --fetch --stacktrace --generator @neo4j/developer-site-generator netlify-training.yml",
+    "build:developer-subfolders": "antora --fetch --stacktrace netlify-developer-subfolders.yml",
     "lint": "npm run lint-xref && npm run lint-asciidoc",
-    "lint-xref": "npm run lint-xref:unversioned && npm run lint-xref:labs-docs",
-    "lint-xref:unversioned": "antora --generator @antora/xref-validator unversioned.yml",
-    "lint-xref:labs-docs": "antora --generator @antora/xref-validator labs-docs.yml",
-    "lint-asciidoc": "./build.sh && ./lint/asciidoctor-errors.sh && ./lint/asciidoctor-warnings.sh"
+    "lint-xref": "npm run lint-xref:developer-subfolders && npm runlint-xref:developer && && npm runlint-xref:labs-docs",
```
Comment (adam-cowley) [inline package.json:17]: One too many &&'s

```
"lint-xref": "npm run lint-xref:developer-subfolders && npm runlint-xref:developer && npm runlint-xref:labs-docs",
```

**Comment by Mogztter** [comment]
@adam-cowley I think we shouldn't add playbooks in https://github.com/neo4j-documentation/docs-ui.
If we want to preview UI changes on the actual content we can do one of the following:


#### Local preview

1. build the ui-bundle.zip (from a local copy of the `docs-ui` repository)
1. use the `--ui-bundle-url` option to point at the locally built `ui-bundle.zip`

```
./node_modules/.bin/antora --stacktrace --generator @neo4j/developer-site-generator --ui-bundle-url=/path/to/local/ui-bundle.zip netlify-training.yml
```

#### Netlify preview

1. put a tag on docs-ui with a prerelease version (i.e. `git tag v1.0.0-alpha.1`)
   It will publish the UI bndle on S3.
2. update the playbooks to use this prerelease version
3. open a pull request and review changes using the deploy preview on Netlify

**Comment by adam-cowley** [comment]
Yeah I agree, keep the UI code and playbooks separate.  Most content previews will be on the latest version of the bundle.  If we're working on a UI change then I think it's reasonable to add the UI bundle path to the antora command.

**Comment by Mogztter** [comment]
> Yeah I agree, keep the UI code and playbooks separate. Most content previews will be on the latest version of the bundle. If we're working on a UI change then I think it's reasonable to add the UI bundle path to the antora command.

I think we are ready, I make sure that the workflow is smooth.
The release process of `docs-ui` is pretty straightforward: `git tag v1.2.3 && git push origin master --tags`

As mentioned above, it's also relatively easy to setup local preview using `--ui-bundle-url`.

I was also impressed by the `gulp preview` on `docs-ui`. It's not perfect but it's pretty useful to quickly iterate on the design.

We will probably need to update the documentation to explain the new architecture so contributors quickly get the big picture.

**Pull request merged**